python/examples/structured_outputs/calls/async_context_stream.py

```
@@ -31,13 +31,13 @@ async def main():
     ctx = llm.Context(deps=library)
     stream: llm.AsyncStream[Book] = await recommend_book.stream_async(ctx, "fantasy")
     async for _ in stream:
-        partial_book: Book | None = None
+        partial_book: llm.Partial[Book] | None = None
         with contextlib.suppress(Exception):
```

we shouldn't be suppressing exceptions here. with `Partial`, if there's an exception, it's likely a legit validation error